x/liquidstake/keeper/rebalancing.go

```diff
@@ -116,15 +115,20 @@ func (k Keeper) Rebalance(
 		if err != nil {
 			redelegation.Error = err
 			failCount++
+
+			k.Logger(ctx).Error(
+				"redelegation failed",
+				types.DelegatorKeyVal, proxyAcc.String(),
+				types.SrcValidatorKeyVal, maxVal.OperatorAddress,
+				types.DstValidatorKeyVal, minVal.OperatorAddress,
+				types.AmountKeyVal, amountNeeded.String(),
+				types.ErrorKeyVal, err.Error(),
+			)
 		}
 
 		redelegations = append(redelegations, redelegation)
 	}
 
-	if failCount > 0 {
```

Comment:
Might be good to keep this one to know when redelegations could not happen.

Reply:
I replaced it with the log above so we get a log for every failed redelegation. instead of having a log for all of them.